sway-lib-std/src/hash.sw

```diff
@@ -297,8 +337,11 @@ impl<T> Hash for [T; 10] where T: Hash {
 ///     assert(result = 0xa80f942f4112036dfc2da86daf6d2ef6ede3164dd56d1000eb82fa87c992450f);
 /// }
 /// ```
-#![inline(never)]
-pub fn sha256<T>(s: T) -> b256 where T: Hash {
+#[inline(never)]
+pub fn sha256<T>(s: T) -> b256
+where
```

Comment:
here is an example of what the `where` clause should look like for `impl` blocks by default